product_pack/tests/test_product_pack.py

```diff
@@ -117,20 +117,3 @@ def test_pack_modifiable(self):
         pack.pack_type = "detailed"
         pack.pack_component_price = "totalized"
         self.assertTrue(pack.pack_modifiable_invisible)
-
-    def test_price_compute_with_pricelist_context(self):
```

Comment:
Why this removal?

Reply:
That test was added in this commit d6044f1 basically to check if we are handling well the pricelist context in the [price_compute](https://github.com/OCA/product-pack/blob/3e8eb93b8431b5808aab2174b21ae357e70145f0/product_pack/models/product_product.py#L33) method. With this refactor we don’t need to handle the pricelist context since we are calling the odoo method “_get_contextual_price()” that does that for us (please check method _pack_line_price_compute()).